docs/spaces/index.asciidoc

```diff
@@ -101,20 +101,7 @@ image::spaces/images/spaces-roles.png["Controlling features visiblity"]
 [[spaces-moving-objects]]
 === Move saved objects between spaces
 
-To <<managing-saved-objects-copy-to-space, copy objects>> from one space to another, open the main menu,
-then click *Stack Management > Saved Objects*.
-
-Alternately, you can move objects using {kib}'s <<managing-saved-objects-export-objects, import and export>>
-interface.
-
-. Navigate to the space that contains your saved objects.
-. Export your saved objects.
-. Navigate to the space where you want to import the objects.
-. Import your saved objects.
-. (Optional) Delete objects in the export space that you no longer need.
-
-{kib} also has beta <<saved-objects-api-import, import>> and
-<<saved-objects-api-export, export>> APIs if you want to automate this process.
+To move saved objects between spaces, you can <<managing-saved-objects-copy-to-space, copy objects>>, or <<managing-saved-objects-export-objects, export and import objects>>.
```

Comment:
I think it's a good idea to mention in the Spaces docs that we can move saved objects, but the Saved Objects Management docs should be the source of truth for this. I think links to the relevant sub-sections on that page is the right call here.